docs/source/modules/zos_blockinfile.rst

```diff
@@ -244,6 +246,36 @@ Examples
              LIB('{{ DB2RUN }}.RUNLIB.LOAD')
        indentation: 16
 
+   - name: Set facts for the following two tasks.
+     set_fact:
+       HLQ: 'ANSIBLE'
+       MLQ: 'MEMBER'
+       LLQ: 'TEST'
+       MEM: '(JCL)'
+       MSG: 'your first JCL program'
+       CONTENT: "{{ lookup('file', 'files/content.txt') }}"
+
+   - name: Update JCL in a PDS member with Jinja2 variable syntax.
+     zos_blockinfile:
+       src: "{{ HLQ }}.{{MLQ}}.{{LLQ}}{{MEM}}"
+       insertafter: "HELLO, WORLD"
+       marker: "//* {mark} *//"
+       marker_begin: "Begin Ansible Block Insertion 1"
```

Comment:
It it my eyesight?  263: "marker_begin" looks like it is in italics, compared to 262: "marker".

Reply:
Yeah , It does look that way but for what ever reason it is being shown as italicized but the result is fine , screen shot below:
<img width="753" alt="image" src="https://user-images.githubusercontent.com/25803172/230650386-269a89ff-6d8c-437c-9266-4588de7692cb.png">